pages/index.vue

```diff
@@ -50,34 +39,50 @@
         preview: query.mode === 'preview'
       };
 
-      return app.$contentful.query('browsePage', variables)
```

Comment:
Since this looks like it was the only place the 'browsePage' query was called, I think that query can be removed from the codebase.